src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6067,4 +6082,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_GeneratorFailedDuringInitialization_Title" xml:space="preserve">
     <value>Generator failed to initialize.</value>
   </data>
+  <data name="ERR_AssignmentInitOnly" xml:space="preserve">
+    <value>Init-only member '{0}' can only be assigned from a constructor, object initialization or 'with' expression of that type.</value>
```

Comment:
>from a constructor, object initialization or 'with' expression of that type [](start = 55, length = 75)

The spec also says: "Inside the init accessor of any property, on this or base"

Reply:
The full quote:
An instance property containing an init accessor is considered settable in the following circumstances:
- During an object initializer
- Inside an instance constructor of the containing or derived type, on this or base
- Inside the init accessor of any property, on this or base

The spec also doesn't mention 'with' expression. I suggest we drop that part for now and make appropriate changes later when the interaction between the features is specified and being implemented.  

---
In reply to: [415097435](https://github.com/dotnet/roslyn/pull/43275#discussion_r415097435) [](ancestors = 415097435)